tests/DomCrawler/Field/ChoiceFormFieldTest.php

```diff
@@ -22,6 +23,7 @@
  */
 class ChoiceFormFieldTest extends TestCase
 {
+    #[DataProvider('clientFactoryProvider')]
```

Comment:
If the goal is to be compatible with PHPUnit 11+, we should also add a job using it.